demo/query_bad/query_bad.md

```diff
@@ -169,18 +167,22 @@ GUAC's external sources (such as Deps.dev). Therefore, if you are seeing less
 than what is shown here, you may want to check back again by re-running the CLI
 in a few minutes!
 
-<img width="1267" alt="6" src="https://user-images.githubusercontent.com/88045217/236021098-184e5bd7-a90d-4509-842c-b6e6988c320d.png">
+<img width="1267" alt="6" src="https://github.com/guacsec/guac/assets/88045217/bd73a5fe-ea6c-46c1-8c56-c52de6c8a567">
```

Comment:
kind of curious , does this graph look different with the additional deps.dev data as well?

Reply:
It does not but we have another demo showing that piece off. I can remove the above snippet that says to check back again if deps.dev is run.

Reply:
This example shows the effect of various images that need to be updated based on a simple library being compromised.